Claudia/src/main/java/FirstTask/Month.java

```diff
@@ -0,0 +1,45 @@
+package FirstTask;
+
+public class Month {
+    public static void main(String[] args) {
+        int month = 9;
```

Comment:
What will be the output for month = 45?

Reply:
Nothing will be displayed for month = 45. I will fix the code. Thanks!

Reply:
Done